xml/poc-map-ui-steps.xml

```diff
@@ -0,0 +1,949 @@
+        <step>
+          <para>
+            On the right, in the <guilabel>Edit Network</guilabel> area,
+            enter the following:
+          </para>
+         <screen>
+           VLAN ID: (accept default)
+           CIDR: 192.168.201.0/24
+           Addresses: (leave empty)
+           Gateway IP: 192.168.201.1
+           Network Groups: EXTERNAL-API
+           Tagged VLAN: (unchecked)
+         </screen>
+        </step>
+        <step>
+          <para>
+            Click <guilabel>Save</guilabel>.
+          </para>
+        </step>
+        <step>
+          <para>
+            On the right side of the <literal>EXTERNAL-VM-NET</literal>
+            network click the <guilabel>edit</guilabel> button.
+          </para>
+        </step>
+          <step>
+            <para>
+              On the right, in the <guilabel>Edit Network</guilabel> area,
+              enter the following:
+            </para>
+           <screen>
+             VLAN ID: (accept default)
+             CIDR: (leave empty)
+             Addresses: (leave empty)
+             Gateway IP: (leave empty)
+             Network Groups: EXTERNAL-VM
+             Tagged VLAN: (unchecked)
+           </screen>
+          </step>
+          <step>
+            <para>
+              Click <guilabel>Save</guilabel>.
+            </para>
+          </step>
+          <step>
+            <para>
+              On the right side of the <literal>GUEST-NET</literal>
+              network click the <guilabel>edit</guilabel> button.
+            </para>
+          </step>
+            <step>
+              <para>
+                On the right, in the <guilabel>Edit Network</guilabel> area,
+                enter the following:
+              </para>
+             <screen>
+               VLAN ID: (accept default)
+               CIDR: 192.168.203.0/24
+               Addresses: (leave empty)
+               Gateway IP: 192.168.203.1
+               Network Groups: GUEST
+               Tagged VLAN: (unchecked)
+             </screen>
+            </step>
+            <step>
+              <para>
+                Click <guilabel>Save</guilabel>.
+              </para>
+            </step>
+            <step>
+              <para>
+                Close the <guilabel>Manage Cloud Settings</guilabel> window.
+              </para>
+            </step>
+    </procedure>
+</section>
+
+
+<section xml:id="deploy-sles">
+  <title>Deploy SLES on the Cloud Servers</title>
+  <para>
+    This exercise deploys SLES on the cloud node servers.
+  </para>
+ <procedure><title>Manage Subnet and Netmask</title>
+   <step>
+     <para>
+       On the forth screen (<guilabel>Add Servers and Assign
+       Server Roles</guilabel>), in the top right, click
+       <guilabel>Manage Subnet and Netmask</guilabel>.
+     </para>
+   </step>
+   <step>
+     <para>
+       On the <guilabel>Manage Subnet and Netmask</guilabel> screen enter
+       the values for the management network:
```

Comment:
note! with a recent Cloud 9 update, this is going to change from 2 separate values to a single CIDR... reach out to @GarySmith or @eapodaca for details (or see https://jira.suse.com/browse/SOC-10251 )

Reply:
Thanks!

Reply:
This might have to be a subsequent update once the change is implemented :) I've noted this down!